### PR TITLE
Adding Acoustic, Electro Library

### PR DESCRIPTION
**Pull request by SirBothersome**
So here it is, twenty or so acoustic samples from @unfa and me. I've tested them with a focus group to make sure that they are of good quality. If it so pleases the court, I can post an archive of the library was well if you want to listen to them yourself. For my next act, I shall try making EDM drum samples.
I may add some toms at a later date if I can get some good samples, bu as of now, I am leaving them out.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
 
-Subaqueous
-  <isaac/at/subaqueousmusic/dot/com>
-  Acoustic and percussion samples
 
-Deficio
-  <deficio/at/live/dot/se>
-  Riser samples
+Antonio "SirBothersome" Alberto
+<https://Soundcloud.com/mrlmmsguy>
```
Comment (tresf) [inline AUTHORS]: Please remove home page urls and soundcloud urls.

Comment (tresf) [inline AUTHORS]: ... and replace with email addresses and ifyou'd like github `@handles`.

**Comment by tresf** [comment]
These need to be consistently named.

e.g. NOT `File 1.flac` and `My Sound2.flac` and `SomeSample1.flac`

**Comment by tresf** [comment]
` Main sample pack/Acoustic/fingersnap1.flac`

**Comment by SirBothersome** [comment]
@tresf, fixed the lowercasing, anything else I missed?

> For my next act, I shall try making EDM drum samples.

I wonder if @StakeoutPunch would want to help...

**Comment by tresf** [comment]
"Clang Clap" "FingerSnap" still inconsistent.

**Comment by SirBothersome** [comment]
Curses... lemme fix that

**Comment by tresf** [comment]
why `space chirp.flac` not match?  Why `Crash1` but `Clap` no number?

**Comment by SirBothersome** [comment]
> why space chirp.flac not match? Why Crash1 but Clap no number?

I'm going to delete the space chirp actually... And since there was only one clap clap, I didn't see a neeed to add a number

**Comment by tresf** [comment]
There's only one crash too, so I'm not sure your point.

**Comment by tresf** [comment]
`space chirp.flac` is still lowercase.

**Comment by tresf** [comment]
Inside the flac file, why did you list yourself as the author of Unfa's work?

Also, are you `Antonio Alberto`,  `SirBothersome` or `Mayo Staccato` ?

I'm sick of these aliases.  Unless you have a great reason to hide behind an alias, just put people's names, or clobber them all with `LMMS Developers`

**Comment by SirBothersome** [comment]
> Inside the flac file, why did you list yourself as the author of Unfa's work?

@tresf, I don't. Some of the samples I made myself. I altered the metadata to read "LMMS Developers", (which I really should have thought of), however, so that's moot.

**Comment by tresf** [comment]
> Inside the flac file, why did you list yourself as the author of Unfa's work?
> > @tresf, I don't. 

Oh, yes you did.

screen shot 2016-08-12 at 1 04 55 am

> I altered the metadata to read "LMMS Developers"

Thanks.

So this PR is finally ready for review.  I'd like someone like @StakeoutPunch or @Umcaruje to chime in on quality and usability now that we finally have consistency in the file names.

**Comment by SirBothersome** [comment]
> Oh, yes you did.

Oh, I see. I added myself to the authors list because I processed and eq'd the samples. I'm not trying to whine here, but I dug through 2+ Gigabytes of raw source audio, so I felt that I contributed to the samples. 

>So this PR is finally ready for review

Yay! I hope this makes the next release.

**Comment by tresf** [comment]
> Oh, I see. I added myself to the authors list because I processed and eq'd the samples. 

If they CC0 it doesn't really matter, but if this were a book, the editor doesn't get to put his name on the cover.

>  I dug through 2+ Gigabytes of raw source audio

It's part of the gig.  This software is a lot of work.  If the samples were easy to do, they would have been done already.  I've seen people spend months of hard work on a single bug and still not have it merged, so let's just proceed with what benefits the project and not start weighing our personal investments.

> Yay! I hope this makes the next release.

No, it will not.

**Comment by SirBothersome** [comment]
> If they're CC0 it doesn't really matter, but if this were a book, the editor doesn't get to put his name on the cover.

It's part of the gig. This software is a lot of work. If the samples were easy to do, they would have been done already. I've seen people spend months of hard work on a single bug and still not have it merged, so let's just proceed with what benefits the project and not start weighing our personal investments.

Artistic arguments aside, it now says "LMMS Developers" in the metadata. A much better idea than watermarking every single one

> No, it will not. (unless the other admins believe otherwise of course)

Out of curiosity, what _would_  have to be done to have it ship with 1.2 in your opinion? I've made some decent progress with the electronic drum samples, and I don't think retrofitting and trimming the demo library will take very long

**Comment by tresf** [comment]
> what would have to be done to have it ship with 1.2 in your opinion?

Well, we have to decide when we cut off the old sample library and roll out the new sample library right?  Or is this proposal meant to be in-addition-to the current library?  e.g. Ok you added some samples, but does that make them the new samples?

You tell us.  What do we do with 27 new sample files?  Do people that download LMMS allow old projects to break because the sample pack was swapped out without notice?

**Comment by SirBothersome** [comment]
> Well, we have to decide when we cut off the old sample library and roll out the new sample library right? Or is this proposal meant to be in-addition-to the current library? e.g. Ok you added some samples, but does that make them the new samples?

You tell us. What do we do with 27 new sample files? Do people that download LMMS allow old projects to break because the sample pack was swapped out without notice?

I was making these with the intention of them replacing the old samples, (part of the reason I posted 
#24 ) I wanted there to be no shadow of doubt as to LMMS' reputation and also, people have been complaining about the samples for a while now, it seems.

> Do people that download LMMS allow old projects to break because the sample pack was swapped out without notice?

I'm sre we're going to arrive at a point where there's going to have to be some sweeping change,  I'm sure an announcement could be added to the LSP and the forum, but all things considered, retrofitting is not that hard, and I doubt the old samples will be missed

**Comment by tresf** [comment]
> retrofitting is not that hard, and I doubt the old samples will be missed

From observation, people do use the base samples projects.  The LSP is full of 'em, so this statement is incorrect.

> I was making these with the intention of them replacing the old samples

We all want that.  We may want a house to replace our current house, but that doesn't mean you push the old one out with a bulldozer without proper warning.

**Comment by SirBothersome** [comment]
> but that doesn't mean you push the old one out with a bulldozer without proper warning.

Hrmm... At any rate, merging this won't change LMMS's master. Also there are  projects that _don't_ have native samples, so the "emptyness" effect probably is probably not new to LSPers. I think putting an announcement in LSP about the upcoming "data apocalypse" should do the trick. I'm going poll my focus group on this.

**Comment by tresf** [comment]
> Also there are projects that don't have native samples, so the "emptyness" effect probably is probably not new to LSPers

Then how the **** would you play it?  This conversation is going nowhere.  This pack needs to be reviewed, that's the next step.  It won't make the 1.2 release unless YOU HELP PLAN it.  We don't do that with a club, we do that with a gentle hand.  We have 500,000 users to worry about.  Don't be naive, it helps no one..

**Comment by SirBothersome** [comment]
>It won't make the 1.2 release unless YOU HELP PLAN it.

Okay, so there are two best options for the release (IMO) are these:
1) Expunge the library, but offer the legacy sounds in a separate download. 
Pros: Easy, minimal setup required, 
Cons: Plenty of room for operator error, 
Also, this really depends how certain devel is of whether or not our legacy library is incorrectly licensed. If the general consensus is that it is not, it would hardly seem ethical or wise to distribute the legacy samples with that knowledge.
2) code an exception routine that loads upon detecting project files made by legacy distros. eg. (pardon my pseudocode) 
`if (kick.ogg == false. 
load "SimilarKick.flac"
continue; //like a switch statement methinks, I forget the proper command for this one
if (snaremuffled.ogg == false)
 load "BetterSnare.flac"
...
break;
FX = true` 
return "YAY_SAMPLESLOADED";
pros: clean, requires no extra download or user action.
cons: probably wouldn't cover all 1000+ samples, more work for devel
probably not worth the effort, as it would likely occupy some 400+ lines of code

These are the best two options I can think of at the moment

**Comment by tresf** [comment]
> it would likely occupy some 400+ lines of code

Do you feel 200+ samples we include now should all be replaced with these 25 you've chosen?

**Comment by SirBothersome** [comment]
No! this is the "Acoustic Library"

**Comment by tresf** [comment]
Where do you suggest we put it exactly?

https://github.com/LMMS/lmms/tree/master/data/samples

How do we keep track of it to know it's part of our new CC0 library initiative?

**Comment by SirBothersome** [comment]
> How do we keep track of it to know it's part of our new CC0 library initiative?

Well, for one thing, these are all flacs. Unless I'm seriously misunderstanding something, I thought this current repo was serving as a holding point for the new samples until integration could be reached

**Comment by tresf** [comment]
> I thought this current repo was serving as a holding point for the new samples until integration could be reached

Exactly, and since 1.2 integration was brought up, it's being discussed.

**Comment by SirBothersome** [comment]
> Where do you suggest we put it exactly?

 Well if we do put it with lmms/data, we could just uppercase the new files to differentiate them

**Comment by SirBothersome** [comment]
Surprise! I'm not dead. I've made some new samples, and have some more in the pipeline.  @tresf , just let me know if I need to stop it already. Oh, and I got rid of the Mhak Kicks, I don't think they were legally cleared, and believe that not many of you liked them

**Comment by SirBothersome** [comment]
Which is hindsight ain't that terrific of an idea... assuming this is even considered a priority

**Comment by SirBothersome** [comment]
Sooo, I put the Mhak kicks back. And... Oh, not much else. @tresf Last update was August 30, but apparently, nobody has taken it upon themselves to look at it, or even comment. ;-;

**Comment by SirBothersome** [comment]
Closing. This PR has gone rather stale, and I think integrating individual libraries will be more efficient at this point. I've also committed this all on my master, which is a problem

**Pull request closed without merging**